gstruct/fields.go

```diff
@@ -14,10 +14,21 @@ import (
 
 //MatchAllFields succeeds if every field of a struct matches the field matcher associated with
 //it, and every element matcher is matched.
-//  Expect([]string{"a", "b"}).To(MatchAllFields(gstruct.Fields{
-//      "a": BeEqual("a"),
-//      "b": BeEqual("b"),
-//  })
+//    actual := struct{
+//      A int
+//      B []bool
+//      C string
+//    }{
+//      A: 5,
+//      B: []bool{true, false},
+//      C: "foo",
+//    }
+//
+//    Expect(actual).To(MatchAllFields(Fields{
```

Comment:
I figure if people know how to import the matches to the top-level namespace, they can probably deduce how to get `Elements` there too by importing the `gstruct` pkg, but I can change this if needed. Again, going for copy/paste-ability